en/option/component/visual-map-piecewise.md

```diff
@@ -118,14 +118,16 @@ This option works when `type` is `piecewise` and `min`/`max`/`splitNumber` are s
 
 If it is set as `true`, an extra piece labeled with "> max" will show.
 
-## selectedMode(string) = 'multiple'
+## selectedMode(string|boolean) = 'multiple'
```

Comment:
Can we add a version hint for the new value type `boolean`?

Reply:
Right. added!